assets/css/planning.scss

```diff
@@ -2,8 +2,18 @@
 
 $tableBoxSize: 40px;
 
+.planning-actions-container-wrapper {
+  width: 100vw;
```

Comment:
This line forces the page to always have an horizontal scrollbar event with only one displayed day